c3.php

```diff
@@ -75,6 +75,7 @@ class_alias('SebastianBergmann\CodeCoverage\Report\Text', 'PHP_CodeCoverage_Repo
     class_alias('SebastianBergmann\CodeCoverage\Report\PHP', 'PHP_CodeCoverage_Report_PHP');
     class_alias('SebastianBergmann\CodeCoverage\Report\Clover', 'PHP_CodeCoverage_Report_Clover');
     class_alias('SebastianBergmann\CodeCoverage\Report\Crap4j', 'PHP_CodeCoverage_Report_Crap4j');
+	class_alias('SebastianBergmann\CodeCoverage\Report\Cobertura', 'PHP_CodeCoverage_Report_Cobertura');
```

Comment:
This alias is not necessary, because `PHP_CodeCoverage_Report_Cobertura` class has never existed.

Reply:
Okay, I've removed the alias